src/frameworks/opengles/gles1_native.rs

```diff
@@ -545,4 +548,7 @@ impl GLES for GLES1Native {
     unsafe fn DeleteRenderbuffersOES(&mut self, n: GLsizei, renderbuffers: *const GLuint) {
         gles11::DeleteRenderbuffersOES(n, renderbuffers)
     }
+    unsafe fn GenerateMipmapOES(&mut self, target: GLenum) {
```

Comment:
Could you move this to the textures section? (Also in all the other files.)

Reply:
Done